#! /bin/sh /usr/share/dpatch/dpatch-run
## -*- diff -*-
## drop-comma-in-configure.dpatch by Rafael Laboissiere <rafael@debian.org>
## 
## DP: Fix syntax error in the configure script

@DPATCH@

--- octave3.1-3.1.50.orig/aclocal.m4
+++ octave3.1-3.1.50/aclocal.m4
@@ -1071,7 +1071,7 @@
     have_opengl_incs=yes, break], [], [
 #ifdef HAVE_WINDOWS_H
 # include <windows.h>
-#endif ]), break], [], [
+#endif ]) break], [], [
 #ifdef HAVE_WINDOWS_H
 # include <windows.h>
 #endif])
